.github/workflows/ci.yml

```
@@ -168,9 +170,23 @@ jobs:
       - name: Frontend typecheck (tsc)
         run: pnpm -C frontend run typecheck
 
+      - name: Frontend CLJS lint
+        env:
+          KNOXX_LINT_BASE_SHA: ${{ github.event.pull_request.base.sha }}
+        run: pnpm -C frontend run lint:cljs:changed
+
+      - name: Frontend migration manifest and ratchets
+        env:
+          KNOXX_MIGRATION_BASE_SHA: ${{ github.event.pull_request.base.sha }}
+          KNOXX_MIGRATION_INFRASTRUCTURE: ${{ contains(github.event.pull_request.body || '', 'Migration infrastructure: yes') }}
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)